.github/actions/bump-skip/action.yml

```diff
@@ -0,0 +1,28 @@
+# This action must be done after the checkout action
+name: 'bump-skip'
+description: 'Set skip-out when we are doing a version bump'
+author: 'dd'
+inputs:
+  event-name:
+    description: 'github.event_name from the calling workflow'
+    required: true
+outputs:
+  is-bump:
+    description: 'yes if this is a push made by bumper; no if it is a regular push'
+    value: ${{ steps.bumptest.outputs.is-bump }}
+runs:
+  using: "composite"
+  steps:
+    - name: Bump test
+      id: bumptest
+      run: |
+        log=$(git log --pretty='%B')
+        echo "log=$log"
+        pattern="^bump .*"
```

Comment:
We could generalize the pattern to something like `[skip_tests]` and allow it anywhere in the log line, but I guess that's a bit ugly in the log

Reply:
Agreed. I made another PR. It requires a change to the common bumper action and the a compensating change to our GHAs.